content/posts/newsletter-020/index.md

```diff
@@ -70,6 +70,38 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library & Tooling Updates
 
+### [wgpu]
+
+![voxel bunny on wgpu](wgpu-conservative-bunny.png)
+Voxel Bunny rendering with conservative rasterization
+
+[wgpu] is a [WebGPU] implementation in Rust. It is safe, efficient,
+and portable: can target both native (Vulkan/D3D/Metal) and the Web.
+
+Most progress in April was focused around [WGSL] shaders and validation.
+[naga] has seen a lot of improvements in the SPIR-V and WGSL parsing, as well
+as backend code generation. Most importantly, it now fully validates both
+statements and expressions. No more accidental foot shots from adding vec2
+and vec3 in the shaders!
+
+The last and the biggest (in terms of shader complexity) example - "water"
+has been successfully ported to WGSL :tada:.
```

Comment:
Swapping this out for a literal emoji, as the shorthand syntax is GitHub-specific:
```suggestion
has been successfully ported to WGSL 🎉.
```